docs/pages/enroll-resources/database-access/enroll-google-cloud-databases/alloydb.mdx

```diff
@@ -0,0 +1,307 @@
+---
+title: Database Access with AlloyDB
+sidebar_label: AlloyDB
+description: How to configure Teleport database access with AlloyDB.
+labels:
+ - how-to
+ - zero-trust
+---
+
+(!docs/pages/includes/database-access/db-introduction.mdx dbType="AlloyDB" dbConfigure="with a service account"!)
+
+## How it works
+
+(!docs/pages/includes/database-access/how-it-works/iam.mdx db="AlloyDB" cloud="Google Cloud"!)
+
+![Teleport Architecture for AlloyDB Access](../../../../img/database-access/guides/alloydb/architecture.png)
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- Google Cloud account
+- Command-line client `psql` installed and added to your system's `PATH` environment variable.
+- A host, e.g., a Compute Engine instance, where you will run the Teleport Database
+  Service
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/9. Create a service account for the Teleport Database Service
+
+(!docs/pages/includes/database-access/gcp-create-service-account-for-db-service.mdx!)
+
+### Grant permissions
+
+The Teleport Database Service needs the following permissions to call appropriate Google Cloud APIs:
+
+```ini
+# Used to generate client certificate
+alloydb.clusters.generateClientCertificate
+# Used to fetch connection information
+alloydb.instances.connect
+```
+
+You can create a custom IAM role with these permissions or use the predefined role 
+[`roles/alloydb.client` (Cloud AlloyDB Client)](https://cloud.google.com/alloydb/docs/reference/iam-roles-permissions), 
+which comes with some additional permissions.
```

Comment:
nice. so the service account doesn't need both "AlloyDB Database User"  and "AlloyDB Client"